### PR TITLE
feat: added image component

### PR DESCRIPTION
**Pull request by mansi2011**
image

feat: added image component
- Responsive Image
- Round Image

### CODE REVIEW

**Review by Prabhav-Sharma — APPROVED** [review]

File: Components/Image/Image.css [hunk 1]
```diff
@@ -0,0 +1,15 @@
+.responsive-image{
```
Comment (SanjayJatti) [inline Components/Image/Image.css:1]: You can add object-fit: cover;

**Review by SanjayJatti — APPROVED** [review]

**Comment by spraveenofficial** [comment]
I was have to suggest, what sanjay have suggested. code looks good.

**Comment by spraveenofficial** [comment]
I was have to suggest, what sanjay have suggested. code looks good.

File: Components/Image/Image.css (hunk 1 above)
Comment (mansi2011) [reply Components/Image/Image.css:1]: okay added it

**Pull request merged**